src/routes/index.js

```diff
@@ -1 +1,51 @@
 // ADD CODE HERE
+var express = require('express');
+var app = express()
```

Comment:
change to this to make it work (I think you already did per our bug this morning) `var app = express.Router()`